frontend/src/AppBuilder/RightSideBar/Inspector/Components/DefaultComponent.jsx

```diff
@@ -147,7 +147,9 @@ export const baseComponentProperties = (
           'properties',
           currentState,
           allComponents,
-          darkMode
+          darkMode,
+          '',
+          component.component.properties[property]?.isDisabled ?? false
```

Comment:
we are already getting this in like 142 right? Do we need to pass this again?

Reply:
Makes sense ✅